nemo/collections/common/parts/preprocessing/collections.py

```diff
@@ -1514,13 +1514,16 @@ def __init__(
         for item in manifest.item_iter(manifests_files, parse_func=self.__parse_item_rttm):
             # Training mode
             rttm_labels = []
-            with open(item['rttm_file'], 'r') as f:
-                for index, rttm_line in enumerate(f.readlines()):
-                    rttm = rttm_line.strip().split()
-                    start = round(float(rttm[3]), round_digits)
-                    end = round(float(rttm[4]), round_digits) + round(float(rttm[3]), round_digits)
-                    speaker = rttm[7]
-                    rttm_labels.append('{} {} {}'.format(start, end, speaker))
+            if item['offset'] is None:
+                item['offset'] = 0
+            if item['rttm_file'] is not None and os.path.exists(item['rttm_file']):
```

Comment:
Should we raise an error if item['rttm_file'] is not None and doesn't exist? Otherwise users may not realize there's a problem with their manifest. Users can set item['rttm_file'] to None if they want to do inference.

Reply:
removed the unnecessary part and applied `get_full_path`